packages/wallet-ui/package.json

```diff
@@ -6,19 +6,19 @@
   "license": "(Apache-2.0 OR MIT)",
   "scripts": {
     "allow-scripts": "yarn workspace root allow-scripts",
-    "build": "react-scripts build",
+    "build": "craco build",
```

Comment:
what is the reason to using craco? when every other dapp in consensys is using react-scripts / gasby

Reply:
get-starknet still using 5.14.0